### PR TITLE
Stripe admin ui additional fixes

### PR DESCRIPTION
**Pull request by valdis**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1028,6 +1028,7 @@ en:
       transaction_requested_by_user: "Great news! %{requester} is interested in \"%{listing_title}\" and has authorized payment for this listing."
       you_have_time_to_accept: "You need to accept or reject the request within %{payment_expires_in}. If you accept within this timeframe, the payment will go through. If you don't accept, the request will be automatically rejected and you will not get paid."
       if_you_do_accept: "If you accept the request within this timeframe, you will receive the money immediately to your account."
+      if_you_do_accept_stripe: "If you accept the request within this timeframe, the payment will go through. You will receive the money directly to your bank account after you have delivered %{listing_title} to %{requester}"
```
Comment (sktoiva) [inline config/locales/en.yml]: Should there be a period at the end?

File: config/routes.rb [hunk 2]
```diff
@@ -171,6 +171,7 @@
       get  "/payment_preferences"                     => "payment_preferences#index"
       put  "/payment_preferences"                     => "payment_preferences#update"
       # PayPal Connect
+      get  "/paypal_preferences" => redirect("/%{locale}/admin/payment_preferences")
```
Comment (sktoiva) [inline config/routes.rb]: Will this work now even without the Stripe feature flag? From now on, the payment preferences are always in this URL with or without stripe flag enabled?

Comment (valdis) [inline config/routes.rb]: Yes

File: config/locales/en.yml (hunk 1 above)
Comment (valdis) [inline config/locales/en.yml]: yeah, it probably should :) thanks

**Pull request merged**